### DISCUSSION TITLE
Size of dataset problem

### DISCUSSION THREAD

**Discussion by shrimonmuke0202** [pyg-team/pytorch_geometric#3273] [Q&A]
I have a large dataset that contains 10 million data. But I cannot create data. pt using PyTorch geometric custom data, because my RAM becomes full. How can I handle this problem?

**Comment by wsad1** [comment]
If your data is stored in separate files, then [this](https://pytorch-geometric.readthedocs.io/en/latest/notes/create_dataset.html) tutorial should help.
